src/Sulu/Bundle/AdminBundle/Tests/bootstrap.php

```diff
@@ -14,6 +14,6 @@
     throw new RuntimeException('Install dependencies to run test suite.');
 }
 
-$autoload = require_once $file;
+$autoload = require $file;
```

Comment:
Because PHPUnit includes the autoload file and then composer includes it, only affects us when using `vendor/bin/phpunit`, which we are not - but I think it is a harmless change.